test/evergreen.yml

```diff
@@ -4191,6 +4191,28 @@ tasks:
           test-name: modularity_metrics
           collection: CodeModularityMetrics
 
+  - name: tsan-warning-metrics
+    commands:
+      - command: timeout.update
+        params:
+          exec_timeout_secs: 43200
+          timeout_secs: 43200
+      - func: "get project"
+      - func: "compile wiredtiger"
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/cmake_build"
+          shell: bash
+          script: |
+            set -o errexit
+            set -o verbose
+            ${PREPARE_TEST_ENV}
+
+            # Explicitly configure TSAN options.
+            export TSAN_OPTIONS="$COMMON_SAN_OPTIONS:history_size=7:print_suppressions=0:suppressions=$(git rev-parse --show-toplevel)/test/evergreen/tsan_warnings.supp:log_path=$(git rev-parse --show-toplevel)/tsan_logs"
```

Comment:
We already have `TSAN_OPTIONS` set for `ubuntu2004-tsan` and AFAIS the only difference is that here we set `log_path=`. Could we just expand the common options by this new one and not redefine all of them?

Or since we run all the tests from the python script, I think it would be better to add this argument to TSAN_OPTIONS there